diff -Naru old/build_package_sources.sh new/build_package_sources.sh
--- old/build_package_sources.sh	2016-02-25 17:59:31.016456980 +0900
+++ new/build_package_sources.sh	2016-02-25 18:06:12.016469547 +0900
@@ -9,9 +9,8 @@
 cd /tmp/page-${VERSION}
 ./autogen.sh
 autoreconf
-# build the final archive
+# sync tmp/page to here
 cd - 
-mkdir -p release
-tar -czf release/page-${VERSION}.tar.gz -C /tmp page-${VERSION}
+rsync -ahv --delete /tmp/page-${VERSION}/ .
 rm -r /tmp/page-${VERSION}
 set +x
