diff -Naur a/Makefile b/Makefile
--- a/Makefile	2012-01-09 06:24:22.000000000 +0900
+++ b/Makefile	2015-09-19 21:34:30.136226192 +0900
@@ -3,7 +3,7 @@
 BINARY=pastebincl
 MANSOURCE=$(BINARY).man.1
 MANOUT=$(BINARY).1
-prefix=/usr/
+prefix=/usr
 
 all: $(BINARY)
 
@@ -19,10 +19,10 @@
 
 
 install: $(BINARY)
-	mkdir -p "$(prefix)bin/"
-	mkdir -p "$(prefix)share/man/man1/"
-	cp $(BINARY) $(prefix)bin/
-	chmod +x $(prefix)bin/$(BINARY)
-	cp $(MANSOURCE) $(prefix)share/man/man1/$(MANOUT)
-	gzip $(prefix)share/man/man1/$(MANOUT)
+	mkdir -p "$(prefix)/bin/"
+	mkdir -p "$(prefix)/share/man/man1/"
+	cp $(BINARY) $(prefix)/bin/
+	chmod +x $(prefix)/bin/$(BINARY)
+	cp $(MANSOURCE) $(prefix)/share/man/man1/$(MANOUT)
+	gzip $(prefix)/share/man/man1/$(MANOUT)
 
diff -Naur a/src/data_devcode.h b/src/data_devcode.h
--- a/src/data_devcode.h	2012-01-08 23:52:55.000000000 +0900
+++ b/src/data_devcode.h	2015-09-19 20:59:49.192968541 +0900
@@ -46,7 +46,7 @@
 #ifndef DEF_DATADEVCODE
 #define DEF_DATADEVCODE
 
-#define DEVELOPPER_CODE "" // TODO: Enter your Pastebin Developper Key here
+#define DEVELOPPER_CODE "a4582d751f7b2b5a3121936cbe5e45bf" // TODO: Enter your Pastebin Developper Key here
 
 #endif//DEF_DATADEVCODE
 
