 makemkvgui/src/api_posix.cpp | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/makemkvgui/src/api_posix.cpp b/makemkvgui/src/api_posix.cpp
index 7d137de..93e48c5 100644
--- a/makemkvgui/src/api_posix.cpp
+++ b/makemkvgui/src/api_posix.cpp
@@ -54,8 +54,6 @@ int ApSpawnApp(const char* verstr,const char* AppName,char* response,size_t resp
 
     if (AppName[0]==':')
     {
-        bool app_found = false;
-        const char* const* app_locations = ApGetAppLocations();
         const char* p_env = getenv("MAKEMKVCON");
 
         AppName++;
@@ -63,6 +61,13 @@ int ApSpawnApp(const char* verstr,const char* AppName,char* response,size_t resp
         if (p_env!=NULL)
         {
             strcpy(app_path,p_env);
+            } else {
+			    strcpy(app_path,AppName);
+			}
+		} else {
+		    strcpy(app_path,AppName);
+		}
+/*
             app_found = true;
         } else {
             for (size_t i=0;app_locations[i]!=NULL;i++)
@@ -105,7 +110,7 @@ int ApSpawnApp(const char* verstr,const char* AppName,char* response,size_t resp
         }
         strcpy(p,AppName);
     }
-
+*/
     strcpy(str_guiserver,"guiserver");
     strcpy(str_apver,verstr);
 
